PurchasesCoreSwift/LocalReceiptParsing/DataConverters/ArraySlice_UInt8+Extensions.swift

```diff
@@ -1,11 +1,21 @@
 //
-// Created by Andrés Boedo on 7/29/20.
-// Copyright (c) 2020 Purchases. All rights reserved.
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ArraySlice_UInt8+Extensions.swift
+//
+//  Created by Andrés Boedo on 7/29/20.
 //
 
 import Foundation
 
 extension ArraySlice where Element == UInt8 {
+
     func toUInt() -> UInt64 {
```

Comment:
would it make sense to rename this signature to `toUInt64`?

Reply:
💯